client/src/components/Admin/cities.list/CitiesList.tsx

```diff
@@ -1,18 +1,21 @@
 import axios from "axios";
-import { useState, useEffect } from "react";
+import { useState, useEffect, FC } from "react";
 import {Link} from 'react-router-dom'
 import '../cities.list/cities-list.css'
+import { City } from '../../../types/types'
 
-const CitiesList = () => {
+interface CitiesListProps {}
```

Comment:
Maybe we need to put all interfaces in a separate file if we are using them in many places. If we're using the interface in the one file let's put it near this file.
https://www.tatvasoft.com/blog/reactjs-best-practices/